### PR TITLE
Add tracking to table sorting and load more

### PR DESCRIPTION
**Pull request by danmitchell-**
### Context
Tracking on register table sorting and load more

### Changes proposed in this pull request
Add event tracking to table sorting links

### CODE REVIEW

File: app/views/kaminari/_paginator.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = paginator.render do
-  %ul.pager-items
+  %ul.pager-items{"data-click-events" => "", "data-click-category" => "Register Table", "data-click-action" => "Pagination"}
```
Comment (gidsg) [inline app/views/kaminari/_paginator.html.haml]: does this empty string `"data-click-events" => ""` do anything, can we just remove it?

Comment (danmitchell-) [reply app/views/kaminari/_paginator.html.haml]: Initializes the JS 👍

Comment (gidsg) [inline app/views/kaminari/_paginator.html.haml]: if you use `nil` instead of `""` does it generate the same markup?

Comment (danmitchell-) [reply app/views/kaminari/_paginator.html.haml]: Doesn't render in the DOM if you use nil.

Comment (gidsg) [reply app/views/kaminari/_paginator.html.haml]: how about `"data-click-events" => true`?

**Pull request merged**